### PR TITLE
Explanation of autoscaling algorithm does not match with current k8s implementation

### PR DESCRIPTION
**Pull request by yunbj**
action was changed at https://github.com/kubernetes/kubernetes/commit/d55f037b7d84e61c81a6b8c20606bb06b6eb20f2 pkg/controller/podautoscaler/horizontal.go L278-281

### CODE REVIEW

**Review by irvifa — COMMENTED** [review]
/sig autoscaling

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25480--kubernetes-io-master-staging.netlify.app/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details

**Comment by kbhawkey** [comment]
@fgrzadkowski @jszczepkowski @directxman12,
Would you have time to review these changes? Thanks!

**Comment by kbhawkey** [comment]
Pinging for review
/cc @kubernetes/sig-scalability-pr-reviews

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -155,10 +155,8 @@ used.
 
 If multiple metrics are specified in a HorizontalPodAutoscaler, this
 calculation is done for each metric, and then the largest of the desired
-replica counts is chosen. If any of these metrics cannot be converted
-into a desired replica count (e.g. due to an error fetching the metrics
-from the metrics APIs) and a scale down is suggested by the metrics which
-can be fetched, scaling is skipped. This means that the HPA is still capable
+replica counts is chosen. If all of these metrics cannot be converted
```
Comment (jkaniuk) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:158]: I won't comment if that matches the actual algorithm here, but current phrasing creates confusion - "This means" looked like it matched "scale down" previously, but not anymore. If so - what is the meaning of the last sentence. Also - why example with API call error was removed?

**Comment by wojtek-t** [comment]
@kubernetes/sig-autoscaling-pr-reviews  is the right alias for this

@mwielgus to triage

**Comment by celestehorgan** [comment]
@kubernetes/sig-autoscaling-pr-reviews – any updates on technical review for this?

**Comment by kbhawkey** [comment]
👋 @yunbj .
It looks like this pull request is inactive. I plan to close it.
If you have questions or comments, feel free to reopen this PR.
/close

**Pull request closed without merging**